src/__tests.ts

```diff
@@ -1,3 +1,3 @@
-declare var require: any;
-const context = require.context('./', true, /\.spec\.ts$/);
+declare var noderequire: any;
+const context = noderequire.context('./', true, /\.spec\.ts$/);
```

Comment:
Why change this? are unit tests working after this change?

Reply:
It was causing an issue on `build` because of a duplicate declaration on `node_modules/@types/node/globals.d.ts:222:13`: `declare var require: NodeRequire;`. Unit tests are working though.